src/Employer/Employer.Web/ViewModels/Part1/Location/LocationEditModel.cs

```diff
@@ -6,16 +7,16 @@ namespace Esfa.Recruit.Employer.Web.ViewModels.Part1.Location
     public class LocationEditModel : VacancyRouteModel
     {
         [Required(ErrorMessage = ValidationMessages.LocationPreferenceMessages.SelectionRequired)]
-        public bool? UseOtherLocation { get; set; }
-
+        public string SelectedLocation { get; set; }
         public string AddressLine1 { get; set; }
-
         public string AddressLine2 { get; set; }
-
         public string AddressLine3 { get; set; }
-
         public string AddressLine4 { get; set; }
-
         public string Postcode { get; set; }
+        public List<string> AvailableLocations { get; set; } = new List<string>();
```

Comment:
Don't think you need `AvailableLocations` in the edit model

Reply:
fixed